src/main/kotlin/domain/Board.kt

```diff
@@ -6,6 +6,9 @@ class Board private constructor(val rows: List<Row>) {
 
     val cellCount = rows.first().size * rows.size
```

Comment:
`mineCount`, `cellCount` 는 다음과 같이 바꿔보는건 어떨까요?
```suggestion

    val mineCount: Int
        get() = rows.flatMap { it.cells }.count { it is Mine }

    val cellCount: Int
        get() = rows.first().size * rows.size
```

```kotlin
val mineCount = ~
val cellCount = ~
```
로 선언하게 되면, 실제로 자바 코드에서 변수 선언이 되기 때문입니다.